BackEnd/scripts/generatePlayerStats.ts

```diff
@@ -56,4 +62,16 @@ async function buildStatsForPlayerWithGameFetch(playerPuuid: string) {
   }
 }
 
-rebuildStatsForLeague(DRAFT);
\ No newline at end of file
+// useful for when theres a remake and u need to purge their playerstats stats for a game
+async function deletePlayerStatsForParticipantOfGame(gameId: number, seasonId: number) {
```

Comment:
uh this doesn't seem to get used?

Reply:
Yep, thats intentional. Its in a script file so ill use the function directly when i need it